resources/views/navigation/header-button.blade.php

```diff
@@ -0,0 +1,2 @@
+{{-- This is shown on the first slideover of the mobile menu, you can leave this empty if you don't want it. By default this is a search button --}}
+<x-button.search />
```

Comment:
yes, I'm not sure what you mean by this

Reply:
This component does not have a namespace e.g. `<x-rapidez::button.search />` and thus this will only work when you add this in your project. This component either needs to be added to this package, come from a package in it's dependencies or be removed.

Reply:
I see thanks for the clarification, will leave this blank so you can overwrite it in the project itself :)